lib/authorization.rb

```diff
@@ -53,7 +53,7 @@ def self.matched_policies(subject_id, actions = nil, object_id = nil)
   def self.generate_default_acls(subject, object)
     {
       acls: [
-        {subjects: [subject], actions: ["Project:view", "Project:delete", "Project:user", "Project:policy", "Project:billing", "Project:github"], objects: [object]},
+        {subjects: [subject], actions: ["Project:view", "Project:edit", "Project:delete", "Project:user", "Project:policy", "Project:billing", "Project:github"], objects: [object]},
```

Comment:
We need to backfill this as well. Or can we wait for wildcard permission PR?

Reply:
Yes, we need to backfill or wait for the wildcard. Furkan also requires backfilling. It might be beneficial to directly backfill with the wildcard.

Reply:
Any review for wildcard support 😄 

https://github.com/ubicloud/ubicloud/pull/1174